.editorconfig

```diff
@@ -0,0 +1,16 @@
+# Editor configuration, see https://editorconfig.org
```

Comment:
What prompted the need for this file?

Reply:
We shouldn't need it. I just got it for "free" when looking at the angular templates. I will remove it to avoid making this change too long